bitnami/mlflow/Chart.yaml

```
@@ -43,4 +43,4 @@ name: mlflow
 sources:
 - https://github.com/bitnami/containers/tree/main/bitnami/mlflow
 - https://github.com/mlflow/mlflow
-version: 1.0.6
+version: 1.0.7
```

Just a little suggestion. In these changes users are able to set the dialect driver and use different databases. According to [semver](https://semver.org) the minor version should bumped because a new feature was added.

```suggestion
version: 1.1.0
```

---

I pushed a new revision with the version bumped to 1.3.0